### ISSUE TITLE
Exception in server tick loop: Server Crash

### ISSUE THREAD

**Issue by Whazaaaaa** [BluSunrize/ImmersiveEngineering#1816]
I'm running a Hermitpack server 1.5.1 and I sometimes (multiple times today) get that my server crashes.

Seeing as Immersive Engineering is the first thing I see in the crash log I thought it was because of Immersive Engineering.

Here is the crashlog: http://pastebin.com/auvKcsQ1

FTBserver-1.10.2-12.18.3.2185-universal.jar (aka Forge2185)
ImmersiveEngineering-0.10-52.jar
minecraft_server.1.10.2.jar

It feels like something Between Immersive Engineering and Mekanism but I dont know much about reading this.

If this is the wrong place for this or this crashlog has nothing to do with Immersive Engineering I appologize. but I'm drawing blanks here hoping to find a solution. And maybe a fix for you mod if this happens on other servers aswell.

Thanks

**Comment by BluSunrize** [comment]
Which IE multiblocks do you have in your world?

**Comment by Whazaaaaa** [comment]
I have no clue what people have built on my server. I use the Hermitpack 1.5.1 for 1.10.2 modpack which has multiple mods.

**Comment by Whazaaaaa** [comment]
Might be EnderIO Energy Storage?

**Comment by BluSunrize** [comment]
Wat.
Energy Storage from EnderIO isn't an IE multiblock.
Ask your users what they have built and if any of them tried to move or teleport a multiblock or some bullshit like that. That's basically the only way this stuff should happen.

**Comment by john01dav** [comment]
My server recently crashed with a similar issue. I have attached a screenshot and the crash report. The crash seems to have occurred in an MV connector block (grass block in the screenshot because the connector was removed to prevent further crashes). The crash also occurred as one player wandered near another player's base while it does not occur when the base is loaded due to a player being inside it. As the multiblock in question spans multiple chunks, I think it seems most likely that this is related to part of a multiblock being loaded. 

Immersive Engineering Version: 0.10-52
Forge Version: 12.18.2.2171
Modpack: Direwolf20 1.10 1.1.4

![2016-12-23_18 42 05](https://cloud.githubusercontent.com/assets/3982497/21464877/c5f105b4-c950-11e6-85bc-b79651c05e3c.png)


[crash-2016-12-23_19.03.20-server.txt](https://github.com/BluSunrize/ImmersiveEngineering/files/671632/crash-2016-12-23_19.03.20-server.txt)
